client/src/components/Navbar.js

```diff
@@ -43,6 +43,12 @@ const useStyles = makeStyles((theme) => ({
 
 const Navbar = (props) => {
   const classes = useStyles();
+
+  const handleSwitch = () => {
+    // show original language
+    return
```

Comment:
It's best to keep feature branches clean and only have code related to that feature so we don't accidentally merge in half finished features